### PR TITLE
Acropolis

### PR DESCRIPTION
**Pull request by bwhm**
Needs review. Please priorities.

### CODE REVIEW

File: meetings/README.md [hunk 1]
```diff
@@ -21,23 +21,40 @@ Each meeting which will be archived has a _meeting identifier_, which is just a
...
-- **Agenda:**
+#### Agenda
+
+:one: Item 1.
+
+:two: ....
+
+#### Minutes
+**Started at:** `time`
+**Present:** `person1, ...`
```
Comment (bedeho) [inline meetings/README.md]: Are appearing on same line, should there not be a break?

File: meetings/README.md [hunk 2]
```diff
@@ -21,23 +21,40 @@ Each meeting which will be archived has a _meeting identifier_, which is just a
...
+
+:one: Item 1.
+
+:two: ....
+
+**Other Topics raised:**
```
Comment (bedeho) [inline meetings/README.md]: Lower case T

File: meetings/acropolis/README.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    - Jens (2/3)
+    - Mokhtar (1/3)
+4. `No PRs merged to master (excluding bugfixes and "pioneer") after "Module Test" (ewd)`
+    - Martin: 1/2
+    - Jens: 1/8
+    - Mokhtar: 1/8
+    - Bedeho: 1/8
+    - Alex: 1/8
+
+`* Bedeho to define n, n_m, n_a, n_j, n_b`
```
Comment (bedeho) [inline meetings/acropolis/README.md]: Lets doo that later, still not clear for Athens, so can update when we know more.

File: meetings/acropolis/README.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+- **Minutes**: `Martin`
+- **Participants**:
+ - `Alex`
+ - `Bedeho`
+ - `Jens`
+ - `Martin`
+ - `Mokhtar`
+
+#### Agenda
+:one: For experiences identified in the [launch meeting](#launch-meeting), review proposed user stories suggestions prepared by each lead, and settle on final set of stories.
```
Comment (bedeho) [inline meetings/acropolis/README.md]: Shouldnt the agenda here include what experiences were identified as being impacted? Also, are leads supposed to have prepared this prior to the meeting? or is it all in the meeting?

File: okrs/README.md [hunk 5]
```diff
@@ -112,68 +109,59 @@ Archived OKRs are found in [archive](/archive), below only live OKRs are found.
...
 | 04.06.19 |   NA  |  NA   |  NA   |  NA   |   NA  | **NA** |
 | 18.06.19 |   NA  |  NA   |  NA   |  NA   |   NA  | **NA** |
 | 02.07.19 |   NA  |  NA   |  NA   |  NA   |   NA  | **NA** |
 
 # Release OKRs
-
-## Objective: `Launch Athens network`
-- **Active from:** 04.02.19
-- **KR Measurement Deadline**: 1 week after Athens launch
+### Objective: `Launch Acropolis Network`
```
Comment (bedeho) [inline okrs/README.md]: Shouldnt this be H2 (##), not H3 (###)

File: okrs/README.md [hunk 6]
```diff
@@ -112,68 +109,59 @@ Archived OKRs are found in [archive](/archive), below only live OKRs are found.
 
 | Date     | KR #1 | KR #2 | KR #3 | KR #4 | KR #5 |Comments|
 |:--------:|:-----:|:-----:|:-----:|:-----:|:-----:|:------:|
-| 09.04.19 |   NA  |  NA   |  NA   |  NA   |   NA  | **NA** |
-| 23.04.19 |   NA  |  NA   |  NA   |  NA   |   NA  | **NA** |
+| 23.04.19 |(31) **0.06**  |  (182,318) **0.08**   |  (30,5) **0.01** | **0.14** |   0  | Changed #1 from `ewd` to `n`  |
```
Comment (bedeho) [inline okrs/README.md]: Bump the rating on KR #4 to 0.25, at least, in terms of ewd it is probably even more, as we have established some good patterns that will speed things up.

File: testnets/acropolis/branding/README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Acropolis
+
+# Naming Rational
+We have a tradition of naming our testnets after cities, locations and empires that's been important in the evolution of democratic principles and the rule of law. After `Mesopotamia`, `Sparta` and `Athens`, we realized the next release would be smaller from a users POV. We therefore landed on `Acropolis`, as [The history of the Acropolis of Athens is long, with moments when democracy philosophy and art flourished, leading to its creation.](http://www.acropolisofathens.gr/aoa/).
+
+# Goals
+The major of goals of the Acropolis testnet is the following:
+
+## Technical
+The technical goals of Acropolis is to improve our technical credibility, and add two new features:
```
Comment (bedeho) [inline testnets/acropolis/branding/README.md]: :D drop the first intro line please

File: testnets/acropolis/branding/README.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+- `Specification` of all modules to improve onboarding for contributors, and set a good presedence for the platform.
```
Comment (bedeho) [inline testnets/acropolis/branding/README.md]: I think these texts are primarily going to be used outside of GitHub and markdown, so let's drop the highlighting, sectioning, bullet points and other visual communication indicators, just stick to content.

File: testnets/acropolis/branding/README.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- [Logomark](#logomark)
+
+# Name
+Acropolis
+
+# Naming Rational
+We have a tradition of naming our testnets after cities, locations and empires that's been important in the evolution of democratic principles and the rule of law. After `Mesopotamia`, `Sparta` and `Athens`, we realized the next release would be smaller from a users POV. We therefore landed on `Acropolis`, as [The history of the Acropolis of Athens is long, with moments when democracy philosophy and art flourished, leading to its creation.](http://www.acropolisofathens.gr/aoa/).
+
+# Goals
+The major of goals of the Acropolis testnet is the following:
```
Comment (bedeho) [inline testnets/acropolis/branding/README.md]: No need to restate this, and in general from below, just make the whole thing a smaller flowing description.

File: testnets/acropolis/branding/README.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+- [Goals](#goals)
+    - [Technical](#technical)
+    - [Community](#community)
+- [Logomark](#logomark)
+
+# Name
+Acropolis
+
+# Naming Rational
+We have a tradition of naming our testnets after cities, locations and empires that's been important in the evolution of democratic principles and the rule of law. After `Mesopotamia`, `Sparta` and `Athens`, we realized the next release would be smaller from a users POV. We therefore landed on `Acropolis`, as [The history of the Acropolis of Athens is long, with moments when democracy philosophy and art flourished, leading to its creation.](http://www.acropolisofathens.gr/aoa/).
```
Comment (bedeho) [inline testnets/acropolis/branding/README.md]: This needs to be way shorter and way more about the final reason for Acropolis, basically more messaging, less blood and guts honesty. Perhaps something specific to what it was about Acropolis and its history/place that made sense for us.

File: meetings/README.md (hunk 1 above)
Comment (bwhm) [reply meetings/README.md]: It can be :)

File: meetings/README.md (hunk 2 above)
Comment (bwhm) [reply meetings/README.md]: fixed

File: meetings/acropolis/README.md (hunk 3 above)
Comment (bwhm) [reply meetings/acropolis/README.md]: These are the MOM. I think it's fair to say that we concluded on this being the OKR, although there are some parameters that still needs to be set before the OKR is finalized for the `Release Plan`.

File: meetings/acropolis/README.md (hunk 4 above)
Comment (bwhm) [reply meetings/acropolis/README.md]: > Shouldnt the agenda here include what experiences were identified as being impacted? 

The agenda of the meeting is to discuss what the leads have identified based on the `Launch Meeting`.

> Also, are leads supposed to have prepared this prior to the meeting? or is it all in the meeting?

This was copypasta from the main README of steps in preparing a release plan.
Maybe not explicit enough, but yes, the leads are supposed to prepare these for the meeting. Then we discuss the content and consequences.

I am changing the phrasing in both docs, for you to review.

File: okrs/README.md (hunk 5 above)
Comment (bwhm) [reply okrs/README.md]: done

File: okrs/README.md (hunk 6 above)
Comment (bwhm) [reply okrs/README.md]: Graded based on 11 items in project, 1 done, 1 in progress.

I agree the biggest have been concluded, so 0.25 it is.

File: testnets/acropolis/branding/README.md (hunk 7 above)
Comment (bwhm) [reply testnets/acropolis/branding/README.md]: done.

File: testnets/acropolis/branding/README.md (hunk 8 above)
Comment (bwhm) [reply testnets/acropolis/branding/README.md]: I disagree here. This is all part of setting a precedence for how we plan releases. What you want to take and use for blogs or other communication can be re-formatted.

File: testnets/acropolis/branding/README.md (hunk 9 above)
Comment (bwhm) [reply testnets/acropolis/branding/README.md]: I'll cut it down a bit, but in this particular circumstance, `Acropolis` was chosen as a consequence of being a sub-release of `Athens`.

File: testnets/acropolis/branding/README.md (hunk 10 above)
Comment (bwhm) [reply testnets/acropolis/branding/README.md]: I'll cut it down a bit, but I think your this is exactly what is stated?

**Pull request merged**